aws_lambda_powertools/event_handler/api_gateway.py

```diff
@@ -247,27 +247,27 @@ def __init__(self, proxy_type: Enum = ProxyEventType.APIGatewayProxyEvent, cors:
         self._cors = cors
         self._cors_methods: Set[str] = {"OPTIONS"}
 
-    def get(self, rule: str, cors: bool = False, compress: bool = False, cache_control: str = None):
+    def get(self, rule: str, cors: bool = True, compress: bool = False, cache_control: str = None):
         """Get route decorator with GET `method`"""
         return self.route(rule, "GET", cors, compress, cache_control)
 
-    def post(self, rule: str, cors: bool = False, compress: bool = False, cache_control: str = None):
+    def post(self, rule: str, cors: bool = True, compress: bool = False, cache_control: str = None):
         """Post route decorator with POST `method`"""
         return self.route(rule, "POST", cors, compress, cache_control)
 
-    def put(self, rule: str, cors: bool = False, compress: bool = False, cache_control: str = None):
+    def put(self, rule: str, cors: bool = True, compress: bool = False, cache_control: str = None):
         """Put route decorator with PUT `method`"""
         return self.route(rule, "PUT", cors, compress, cache_control)
 
-    def delete(self, rule: str, cors: bool = False, compress: bool = False, cache_control: str = None):
+    def delete(self, rule: str, cors: bool = True, compress: bool = False, cache_control: str = None):
```

Comment:
@heitorlessa  i don't think we want it like this.

Reply:
@heitorlessa - i think the best we can do is default it to `None`  i can make a PR for this.